test/tests.js

```diff
@@ -355,49 +365,75 @@ describe('Appboy Forwarder', function () {
     });
 
     it('should log a purchase event', function() {
+                                                     mParticle.forwarder.process(
```

Comment:
Something is seriously wrong with the indenting here.

Reply:
yea my IDE is weird as you've seen during pairing sessions. I didn't catch this when committing.  Will be fixed.